README.md

```diff
@@ -1,90 +1,142 @@
                                             'updater_is_active', 'updater_user_tags', 'updater_last_login_at', 'updater_time_zone', 
                                             'updater_organization_id', 'updater_organization_domain_names' , 'updater_organization_organization_tags'
-                                          ]
+                                            ]
 ```
 *Note: This package only integrates the above ticket_field_history_updater_columns values. If you'd like to include additional updater fields, please create an [issue](https://github.com/fivetran/dbt_zendesk/issues) specifying which ones.*
 
-### Changing the Build Schema
-By default this package will build the Zendesk staging models within a schema titled (<target_schema> + `_zendesk_staging`), the Zendesk intermediate models within a schema titled (<target_schema> + `_zendesk_intermediate`), and the Zendesk final models within a schema titled (<target_schema> + `_zendesk`) in your target database. If this is not where you would like your modeled Zendesk data to be written to, add the following configuration to your `dbt_project.yml` file:
+### Extending and Limiting the Ticket Field History
+This package will create a row in `zendesk__ticket_field_history` for each day that a ticket is open, starting at its creation date. A Zendesk ticket cannot be altered after being closed, so its field values will not change after this date. However, you may want to extend a ticket's history past its closure date for easier reporting and visualizing. To do so, add the following configuration to your root `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+config-version: 2
+
+vars:
+  zendesk:
+    ticket_field_history_extension_months: integer_number_of_months # default = 0 
+```
+
+Conversely, you may want to only track the past X years of ticket field history. This could be for cost reasons, or because you have a BigQuery destination and have over 4,000 days (10-11 years) of data, leading to a `too many partitions` error in the package's incremental models. To limit the ticket field history to the most recent X years, add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
 # dbt_project.yml
 
 ...
+config-version: 2
+
+vars:
+  zendesk:
+    ticket_field_history_timeframe_years: integer_number_of_years # default = 50 (everything)
+```
+
+### Changing the Build Schema
+By default this package will build the Zendesk staging models within a schema titled (<target_schema> + `_zendesk_staging`), the Zendesk intermediate models within a schema titled (<target_schema> + `_zendesk_intermediate`), and the Zendesk final models within a schema titled (<target_schema> + `_zendesk`) in your target database. If this is not where you would like your modeled Zendesk data to be written to, add the following configuration to your root `dbt_project.yml` file:
```

Comment:
We should update this to reflect the updates from the source package. In particular we should adjust `_zendesk_staging` to instead be `_zendesk_source`.

Reply:
updated